src/test/spec/initial_dns_seedlist_discovery.rs

```diff
@@ -31,6 +31,7 @@ struct ResolvedOptions {
     ssl: bool,
     load_balanced: Option<bool>,
     direct_connection: Option<bool>,
+    srv_max_hosts: Option<u32>,
```

Comment:
my sync brought in a bunch of tests for srvMaxHosts. this addition was necessary for the test files to be deserialized successfully, though I added logic below to avoid actually running the new tests.

not sure u32 was the right type choice here, but it seemed consistent with other options like this in `ClientOptions`?